addons/binding/org.openhab.binding.enocean/ESH-INF/thing/CentralCommand.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="enocean"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="centralCommand">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="serialbridge" />
+			<bridge-type-ref id="tcpbridge" />
+		</supported-bridge-type-refs>
+
+		<label>A5-38 - Central command</label>
```

Comment:
The labels should be the name of the physical device that has been discovered (or selected to be added).
The names should be user-friendly, i.e. "A5-38 -" should not be here; if relevant to tell two different devices apart, you might add it in brackets to the end, but usually it should suffice in the description.

Reply:
As the EEP number is so commonly used in device specs, I wanted to place it directly in the name/label.

Reply:
How about at least changing the order, e.g. to "Central Command (A5-38)"? Btw, we usually use capitalisation in Labels, so you might want to adapt them.

Reply:
Removed EEP reference from label and added it to the description. Labels use capitalisation now.